src/App.tsx

```diff
@@ -1,9 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+interface Film {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+  imdbId: string
+}
+
 export const App: React.FC = () => {
+  const [movies, setMovies] = useState<Film[]>(moviesFromServer);
```

Comment:
it will be better to not use `state` if there is possible to avoid it. `useMemo` hook will be better here

Reply:
Done